src/main/java/org/folio/innreach/mapper/InnReachTransactionHoldMapper.java

```diff
@@ -10,22 +10,25 @@
 import org.folio.innreach.dto.PatronHoldDTO;
 import org.folio.innreach.dto.TransactionHoldDTO;
 
-@Mapper(componentModel = "spring", injectionStrategy = InjectionStrategy.CONSTRUCTOR, uses = {MappingMethods.class, InnReachTransactionPickupLocationMapper.class})
+@Mapper(componentModel = "spring", injectionStrategy = InjectionStrategy.CONSTRUCTOR, uses = {MappingMethods.class, MappingMethodsToEntity.class,InnReachTransactionPickupLocationMapper.class})
 public interface InnReachTransactionHoldMapper {
 
   @AuditableMapping
   TransactionHoldDTO toItemHoldDTO(TransactionItemHold transactionItemHold);
 
+  @AuditableMappingToEntity
   TransactionItemHold toItemHold(TransactionHoldDTO dto);
```

Comment:
Why would we need to map metadata from dto to entity? We manage these values using JPA auditing
https://github.com/folio-org/mod-inn-reach/blob/e931b68064a8d3da25c28a681e6acf28015c32e3/src/main/java/org/folio/innreach/config/JpaAuditingConfig.java#L17-L19

Reply:
Thanks for information

Reply:
removed